test_build.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+"""Test script for build functionality."""
+
+import os
+import tempfile
+import unittest
+from unittest.mock import patch, MagicMock
+import json
+
+# Import functions from build.py for testing
+from build import get_buymeacoffee_stats, render_index, get_manifest
+
+class TestBuildFunctions(unittest.TestCase):
```

Comment:
Use pytest instead of unittest, and use requests-mock instead of MagicMock if possible please thanks